.github/workflows/test-env-deploy.yml

```diff
@@ -171,9 +171,38 @@ jobs:
         run: |
           echo ${{steps.get_tags.outputs.result}}
 
+  testmo-report-preparation:
+    needs: prepare-tests
+    runs-on: ubuntu-22.04
+    outputs:
+      testmo-run-id: ${{ steps.run-tests.outputs.testmo-run-id }}
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version-file: ".nvmrc"
+          cache: npm
+      - name: install dependencies
+        working-directory: .github/workflows
+        run: npm ci
+      - name: create testmo run
+        working-directory: .github/workflows
+        run: |
+          npx testmo automation:run:create \
```

Comment:
If you install `testmo` in pkg.json why not add `testmo automation:run:create` to `package.json` `scripts` object and here execute `npm run <YOUR_SCRIPT_NAME>`?